File: tourneyAPI/Models/Entities/Characters/Wolf.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Wolf : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Wolf()
    {
        Id = CharacterId.Wolf;
        characterName = CharacterName.WOLF;
        archetype = Archetype.ALL_ROUNDER;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.A;
    }

}

File: tourneyClient/src/models/entities/Characters/Wolf.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Wolf: Character =
{
    id: CharacterId.Wolf,
    characterName: CharacterName.WOLF,
    archetype: Archetype.ALL_ROUNDER,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.A
}

export default Wolf; 